### PR TITLE
add more test

### PR DESCRIPTION
**Pull request by gucong3000**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -63,19 +63,25 @@ module.exports = postcss.plugin( 'postcss-flexboxfixer', function( opts ) {
...
+        for (var j = startStep; j < args.length; j++) {
             var position, color, colorIndex;
             if(args[j].name === 'color-stop'){
                 position = args[j].args[0].name;
-                colorIndex = 1;
+                if (args[j].args[1]) {
+                    colorIndex = 1;
+                } else {
+                    colorIndex = 0;
+                    position = (j - startStep) / (args.length - startStep - 1);
```
Comment (gucong3000) [inline index.js:70]: in this case:

``` CSS
.direction {
    background: -webkit-gradient(linear, left top, right bottom, from(black), color-stop(rgba(0, 0, 0, 0.5)), to(white));
}
```

`colorIndex` should be `0`.
if not, this line will throw a Error

``` JavaScript
color = args[j].args[colorIndex].name;
```

File: index.js [hunk 2]
```diff
@@ -222,8 +228,8 @@ module.exports = postcss.plugin( 'postcss-flexboxfixer', function( opts ) {
     }
 
     return function( css ) {
-        css.walkDecls( /(^background$)/, function( decl ) {
-            if(!/-webkit(-\w*|)-gradient/.test(decl.value)){
+        css.walkDecls(function(decl) {
```
Comment (gucong3000) [inline index.js:226]: all decl that can use value `url(...)`, that can be replace with `-webkit-gradient`

File: index.js [hunk 3]
```diff
@@ -222,8 +228,8 @@ module.exports = postcss.plugin( 'postcss-flexboxfixer', function( opts ) {
...
+            if (!/-webkit-gradient/.test(decl.value)) {
```
Comment (hallvors) [inline index.js]: This change means we stop handling -webkit-linear-gradient and -webkit-radial-gradient - right? Is this on purpose?

Comment (gucong3000) [inline index.js]: @hallvors It's not stop stop handling, it is handling every `prop`, not just `background`. like: `background-image`, `list-style-image`, `mask`

Comment (hallvors) [inline index.js]: @gucong3000 Yes, I get that :) I'm asking about changing 

```
/-webkit(-\w*|)-gradient/
```

to

```
/-webkit-gradient/
```

The former is meant to handle CSS using any of `-webkit-gradient`, `-webkit-linear-gradient`, or `-webkit-radial-gradient`. Did you decide to remove support for those two older forms, or was it an accident?

Comment (gucong3000) [inline index.js]: Oh, I was wrong, I'll change it.

Comment (gucong3000) [inline index.js]: fixed, and I added 3 tests for this case.

**Comment by gucong3000** [comment]
@hallvors All done.

**Comment by hallvors** [comment]
Thanks!

**Pull request merged**